src/atc/atc-info.class.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export enum AtcType {
+  unknow,
+  delivery,
+  ground,
+  tower,
+  departure,
+  approach,
+  radar
```

Comment:
These should be upper case and has to be changed in multiple places.
`Unknown` is misspelled.